### PR TITLE
create-pop-up-windows-project-details-desktop-mobile-version-JS

### PR DESCRIPTION
**Pull request by evillaz**
This branch's purpose was to:

- Create pop-up windows to display the project details on each one.
- Changed the project to fulfill the hover status and show the "See Project" button
- Updated CSS, HTML, and JS to correct linters and fulfill the requirements.
*corrected modals styling
*created js code to load main projects section

### CODE REVIEW

File: My-Portfolio/css/style.css [hunk 1]
```diff
@@ -281,6 +274,120 @@ ul {
     background-repeat: no-repeat;
     background-position: center;
   }
+
+  .project-preview {
```
Comment (GedeonTS) [inline My-Portfolio/css/style.css:394]: - Kindly make sure your Popup stick to the Figma design as per the project requirements, by:

- [ ] on the desktop kindly make sure to respect the margins, as you can see below the details popup doesn't touch the edges of the screen, there is a dark blur overlay behind.

- Figma:
Screenshot from 2023-01-31 22-26-26
- Your design:
Screenshot from 2023-01-31 22-26-12


- [ ] on mobile,  Kindly put make sure kindly make sure we can scroll the whole screen(popup +overlay) as the details popup must display as a whole.
- Figma:
Screenshot from 2023-01-31 22-26-46
- Your design:

[Screencast from 01-31-2023 10:27:10 PM.webm](https://user-images.githubusercontent.com/97834160/215878333-1e6e1e42-3451-4d2a-a3f0-617d7c45c39a.webm)

File: My-Portfolio/index.html [hunk 2]
```diff
@@ -8,572 +8,612 @@
...
+                </li>
+              </ul>
+              <div id="cardButton">
+                <button data-name="card1" class="button projectButton" type="button">
+                  <span id="buttonText">See Project</span>
+                </button>
               </div>
-              <a href="#" id="menu-button"></a>
-              <ul class="flex-container" id="deskNavBar">
+            </div>
```
Comment (GedeonTS) [inline My-Portfolio/index.html]: - [ ] As recommended in the project requirements [here](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#:~:text=You%20need%20to%20update%20the%20main%20page%20so%20that%20the%20projects%20section%20is%20created%20dynamically%20using%20the%20information%20stored%20in%20that%20JavaScript%20object.%20Remember%2C%20all%20of%20the%20HTML%20in%20that%20section%20is%20created%20when%20the%20page%20loads.) Kindly make sure the works section is generated dynamically with javascript by looping inside the array of data you have created :recycle:

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :

Hi @evillaz , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :tophat:  :+1: 
- No linter errors :+1: 
- Popup is showing and hiding when the user clicks:+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @evillaz  👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job so implementing the project requirements! 🥇 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉 

## Highlights
✔️professional PR👍 
✔️professional README👍 
✔️No linter Errors 👍 

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**